flatgfa-py/src/lib.rs

```diff
@@ -444,8 +470,131 @@ impl PyPath {
         self.steps().__len__()
     }
 }
+#[derive(Clone)]
+#[pyclass(frozen)]
+#[pyo3(name = "ChunkEvent", module = "flatgfa")]
+struct PyChunkEvent {
+    chunk_event: Arc<ChunkEvent>,
+}
+
+#[pymethods]
+impl PyChunkEvent {
+    #[getter]
+    fn handle(&self) -> String {
+        let seg_num: u32 = self.chunk_event.handle.segment().into();
+        format!("{}, {}", &self.chunk_event.handle.orient(), seg_num)
+    }
+
+    #[getter]
+    fn range(&self) -> String {
+        match self.chunk_event.range {
+            flatgfa::ops::gaf::ChunkRange::None => "".to_string(),
+            flatgfa::ops::gaf::ChunkRange::All => "all".to_string(),
+            flatgfa::ops::gaf::ChunkRange::Partial(start, end) => {
+                format!("[{}, {})", start, end)
+            }
+        }
+    }
+    fn get_seq(&self, gfa: &PyFlatGFA) -> String {
+        let inner_gfa = gfa.0.view();
+        let seq = inner_gfa.get_seq_oriented(self.chunk_event.handle);
+
+        match self.chunk_event.range {
+            flatgfa::ops::gaf::ChunkRange::Partial(start, end) => seq.slice(start..end).to_string(),
+            flatgfa::ops::gaf::ChunkRange::All => seq.to_string(),
+            flatgfa::ops::gaf::ChunkRange::None => "".to_string(),
+        }
+    }
+}
+
+#[pyclass]
+#[pyo3(name = "GAFLine", module = "flatgfa")]
+
+struct PyGAFLine {
+    store: Arc<Store>,
+    vec_chunk: Vec<PyChunkEvent>,
+    gaf: String,
+}
+#[pymethods]
+impl PyGAFLine {
+    #[getter]
+    fn get_name(&self) -> String {
+        self.gaf.clone()
+    }
+
+    #[getter]
+    fn get_chunk_list(&self) -> Vec<PyChunkEvent> {
+        self.vec_chunk.clone()
+    }
+    fn get_sequence(&self) -> String {
```

Comment:
Let's try to be consistent about what we call the method that gets the underlying sequence data from an object. Elsewhere (e.g., in `PySegment`), it's called `sequence()`, so staying consistent with that would make sense.